reddit.js

```diff
@@ -0,0 +1,139 @@
+{
+	"translatorID": "23bacc11-98e3-4b78-b1ef-cc2c9a04b893",
+	"label": "reddit",
+	"creator": "Lukas Kawerau",
+	"target": "^https?://www\\.reddit\\.com",
```

Comment:
Could you make this stricter? I think everything you're looking for is either in `r/comments` or `/search` isn't it?

Reply:
I've left this as is for now, because this way we can select multiple posts from the main feed on reddit.com